usaspending_api/disaster/tests/integration/conftest.py

```diff
@@ -0,0 +1,290 @@
+        obligations_incurred_by_program_object_class_cpe=1000,
+        gross_outlay_amount_by_program_object_class_cpe=10000,
+    )
+    mommy.make(
+        fabpaoc,
+        final_of_fy=True,
+        treasury_account=tas2,
+        submission=sub3,
+        program_activity=pa4,
+        object_class=oc3,
+        disaster_emergency_fund=defc3,
+        obligations_incurred_by_program_object_class_cpe=10000,
+        gross_outlay_amount_by_program_object_class_cpe=1000,
+    )
+    mommy.make(
+        fabpaoc,
+        final_of_fy=True,
+        treasury_account=tas3,
+        submission=sub3,
+        program_activity=pa4,
+        object_class=oc3,
+        disaster_emergency_fund=defc3,
+        obligations_incurred_by_program_object_class_cpe=100000,
+        gross_outlay_amount_by_program_object_class_cpe=100,
+    )
+    mommy.make(
+        fabpaoc,
+        final_of_fy=True,
+        treasury_account=tas3,
+        submission=sub4,
+        program_activity=pa4,
+        object_class=oc4,
+        disaster_emergency_fund=defc4,
+        obligations_incurred_by_program_object_class_cpe=1000000,
+        gross_outlay_amount_by_program_object_class_cpe=10,
+    )
+    mommy.make(
+        fabpaoc,
+        final_of_fy=True,
+        treasury_account=tas3,
+        submission=sub4,
+        program_activity=pa4,
+        object_class=oc4,
+        disaster_emergency_fund=defc4,
+        obligations_incurred_by_program_object_class_cpe=10000000,
+        gross_outlay_amount_by_program_object_class_cpe=1,
+    )
+    mommy.make(
+        fabpaoc,
+        final_of_fy=True,
+        treasury_account=tas4,
+        submission=sub5,
+        program_activity=pa5,
+        object_class=oc5,
+        disaster_emergency_fund=defc5,
+        obligations_incurred_by_program_object_class_cpe=0,
+        gross_outlay_amount_by_program_object_class_cpe=0,
+    )
+    mommy.make(
+        fabpaoc,
+        final_of_fy=True,
+        treasury_account=tas5,
+        submission=sub1,
+        disaster_emergency_fund=None,
+        obligations_incurred_by_program_object_class_cpe=10,
+        gross_outlay_amount_by_program_object_class_cpe=1000000,
+    )
+    mommy.make(
+        fabpaoc,
+        final_of_fy=True,
+        treasury_account=tas6,
+        submission=sub1,
+        disaster_emergency_fund=None,
+        obligations_incurred_by_program_object_class_cpe=100,
+        gross_outlay_amount_by_program_object_class_cpe=100000,
+    )
+
+
+class Helpers:
+    @staticmethod
+    def post_for_count_endpoint(client, url, def_codes=None):
+        if def_codes:
+            request_body = json.dumps({"filter": {"def_codes": def_codes}})
+        else:
+            request_body = json.dumps({"filter": {}})
+        resp = client.post(url, content_type="application/json", data=request_body)
+        return resp
+
+
+@pytest.fixture
+def helpers():
+    return Helpers
+
+
+__all__ = ["disaster_account_data", "helpers"]
```

Comment:
I think we should consider splitting up this file. These are the tests for one endpoint and we're already looking at 300 lines.

Reply:
These aren't the actual tests but fixtures that will be used by multiple tests. For example, the `post_for_count_endpoint()` function will be used by all four count endpoints. Can still break these up and import into conftest.py to break up files and still have the desired functionality.

Reply:
Misspoke, they are the fixtures for the tests of one group. Even then, I fear this growing into a 1000 line file where nobody knows which fixtures go to which test. I would still favor splitting.

Reply:
Split the fixtures into different files and imported them into conftest for the same effect without over cluttering conftest.py.